### PR TITLE
Adda test for an unhandled builtin exception to ExceptionTest.scala

### PR DESCRIPTION
**Pull request by cocreature**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala [hunk 1]
```diff
@@ -66,13 +64,28 @@ class ExceptionTest extends AnyWordSpec with Matchers with TableDrivenPropertyCh
...
+        ),
+      )
 
     val testCases = Table[String, SResult](
       ("expression", "expected"),
       ("M:unhandled1", SResultError(DamlEUnhandledException(e1))),
       ("M:unhandled2", SResultError(DamlEUnhandledException(e1))),
       ("M:unhandled3", SResultError(DamlEUnhandledException(e1))),
       ("M:unhandled4", SResultError(DamlEUnhandledException(e2))),
+      ("M:divZero", SResultError(DamlEUnhandledException(divZeroE))),
```
Comment (remyhaemmerle-da) [inline daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala:88]: This is already tested in  `daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SBuiltinTest.scala` .
One more, in a slightly different context, do not harm.

**Review by remyhaemmerle-da — COMMENTED** [review]
LGTM

File: daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala (hunk 1 above)
Comment (cocreature) [reply daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala:88]: Yeah I saw that. It seemed useful to have one case for this here but also happy to drop it if you prefer.

Comment (remyhaemmerle-da) [reply daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala:88]: :+1:

Comment (remyhaemmerle-da) [reply daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala:88]: meaning you can keep it.

**Review by sofiafaro-da — APPROVED** [review]
Thanks!

**Pull request merged**